spacewalk/config/etc/httpd/conf.d/zz-spacewalk-www.conf

```diff
@@ -63,7 +63,8 @@ ProxyTimeout 600
 <Proxy ajp://localhost:8009>
   ProxySet min=1
 </Proxy>
-RewriteRule ^/rhn/Login2\.do ajp://localhost:8009/rhn/Login.do [P]
+RewriteRule ^/rhn/Login2\.do ajp://localhost:8009/rhn/manager/login [P]
+RewriteRule ^/rhn/(Re)?Login(.*)$ /rhn/manager/login
```

Comment:
Do we still need to keep `Login2.do` and this redirect around? And why do we need a redirect from `Login` / `ReLogin` here?

Reply:
AFAIK Login2.do is used for Kerberos authentication. The Login/Relogin has been added as per the comment above.

Reply:
As mentioned above, I would still question the need of the new redirect (`Login/ReLogin`), but also the existence of `Login2.do` in general. I would be okay to keep it if it is needed for Kerberos authentication, but I would prefer if we properly understand the reason.

Reply:
Ok, removed as per the comment above